### PR TITLE
Rspec errors

### PR DESCRIPTION
**Pull request by jbranham1**
**What did you change?**
Fixed files to pass our spec harness
**Next Steps**

**Known Issues**

### CODE REVIEW

File: lib/item_repository.rb [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
 require 'csv'
 require 'bigdecimal'
+require 'time'
 
 class ItemRepository
   def initialize(filepath)
     @items = build_items(filepath)
   end
 
+  # :nocov:
```
Comment (abreaux26) [inline lib/item_repository.rb:11]: @jbranham1 what is this?

File: lib/item_repository.rb [hunk 2]
```diff
@@ -64,7 +71,7 @@ def create(attributes)
   end
 
   def update(id, attributes)
-    find_by_id(id).update(attributes)
+    find_by_id(id)&.update(attributes)
```
Comment (abreaux26) [inline lib/item_repository.rb:74]: nice!

File: lib/item_repository.rb (hunk 1 above)
Comment (jbranham1) [reply lib/item_repository.rb:11]: @abreaux26 it ignores inspect when you run simplecov!

Comment (jbranham1) [reply lib/item_repository.rb:11]: We may want to ask about this, but for now this is what google told me.

File: lib/item_repository.rb (hunk 2 above)
Comment (aschwartz1) [reply lib/item_repository.rb:74]: What's the ampersand doing there?

Comment (abreaux26) [reply lib/item_repository.rb:74]: @aschwartz1 it is used so you don't have to use
```ruby
if !find_by_id(id).nil?
``` 
or 
```ruby
unless find_by_id(id).nil?
```

**Review by abreaux26 — APPROVED** [review]

File: lib/item_repository.rb (hunk 2 above)
Comment (jbranham1) [reply lib/item_repository.rb:74]: RT^
@aschwartz1 You read it as find by id and if there is an id then update.

Comment (aschwartz1) [reply lib/item_repository.rb:74]: Oh, I see. Neat!

**Review by aschwartz1 — APPROVED** [review]

**Pull request merged**